src/Magnum/OvrIntegration/Compositor.cpp

```diff
@@ -108,8 +103,6 @@ LayerQuad& LayerQuad::setQuadSize(const Vector2& size) {
     return *this;
 }
 
-Compositor::Compositor() = default;
```

Comment:
I would keep the constructor here, the class has heavy members (`std::vector`) and having the constructor inlined will increase compile time and binary size.